src/daily_output.ts

```diff
@@ -20,10 +19,9 @@ export class DailyOutput {
 	private data = this.read();
 	private latestBlock = 0;
 	private latestTimestamp = getLastTimestamp();
-	private bundlesTips: { [txId: string]: number } = {};
-	private pendingSizeSumsOfUnverifiedBundleTips: { [txId: string]: { walletAddress: string; fileSize: number }[] } =
-		{};
+	private bundlesTips: { [txId: string]: { tip: number; size: number; address: string } } = {};
 	private unbundledBundleTxIDs: string[] = [];
+	private bundleFileCount: { [txId: string]: number } = {};
```

Comment:
This will be used at 
- `aggregateData` to track the total number of file metadata transactions
- `finishDataAggregation` to aggregate one single file count if it's yet not unbundled